src/frontend/metrics/generateMetricsReport.ts

```diff
@@ -22,17 +23,25 @@ export default function generateMetricsReport({
       addToSet(countries, positionToCountries(lat, lon));
     }
   }
-
   return {
     type: 'metrics-v1',
     appVersion: packageJson.version,
     os,
     osVersion,
     screen: {width: screen.width, height: screen.height},
     ...(countries.size ? {countries: Array.from(countries)} : {}),
+    percentageOfNetworkAvailability:
+      getPercentageOfNetworkAvailability(observations),
```

Comment:
I'm having a type error here (gh doesn't show it). I won't paste it since its too long, but the gist of it is:
` is 'readonly' and cannot be assigned to the mutable type`

Any clues @EvanHahn ? (probably if you run it locally you'd have a better clue...

Reply:
`getPercentageOfNetworkAvailability`'s argument should change to:

```diff
 export function getPercentageOfNetworkAvailability(
-  observations: ReadonlyArray<Observation>,
+  observations: ReadonlyDeep<Array<Partial<Observation>>>,
 ): number {
```